CMakeLists.txt

```diff
@@ -102,9 +102,15 @@ if(CEF_CMAKE_OS_LINUX)
         debug ${CMAKE_CURRENT_SOURCE_DIR}/${cefName}/Debug/libcef.so
         optimized ${CMAKE_CURRENT_SOURCE_DIR}/${cefName}/Release/libcef.so
     )
+    
+    if ($<CONFIG> STREQUAL "RelWithDebInfo" OR $<CONFIG> STREQUAL "Release" OR $<CONFIG> STREQUAL "MinSizeRel")
```

Comment:
instead of checking three things for release, why not check one thing for debug?